zuul.d/kuttl_multinode.yaml

```diff
@@ -82,6 +82,6 @@
     pre-run:
       - ci/playbooks/kuttl/kuttl-from-operator-deps.yaml
     run:
-      - ci/playbooks/kuttl/run-kuttl-from-operator-targets.yaml
+      - ci/playbooks/kuttl/kuttl-from-operator-run.yaml
```

Comment:
Will this break other jobs in other operators expecting the old behavior?

Reply:
no, only watcher and nova operators use this parent job, and they are both tested with this PR

Reply:
cool then